flowcraft/generator/components/assembly_processing.py

```diff
@@ -229,3 +229,38 @@ def __init__(self, **kwargs):
                 "version": "1.22.0-1"
             }
         }
+
+class Quast(Process):
+    """Assess assembly quality using QUAST
+
+    This process is set with:
+
+        - ``input_type``: assembly
+        - ``output_type``: tsv
+        - ``ptype``: post_assembly
+
+    """
+
+    def __init__(self, **kwargs):
+        super().__init__(**kwargs)
+
+        self.input_type = "fasta"
+        self.output_type = "tsv"
+
+        self.params = {
+            "reference": {
+                "default": "null",
+                "description": "Compare the assembly to this reference genome"
+            },
+            "genomeSize": {
```

Comment:
Is there a reason for the `genomeSize` to be in "BP"? This parameter already occurs in several other components in "MB" units, so for consistency it would be great for this one to be to. Moreover, when pipelines are built with the `--merge-params` option, parameters with the same name for different components are merged into a single one and this could cause problems. If you really need the units to be in BP, I suggest changing the name of the parameter (to `genomeSizeBp` or any other you can think of) to avoid these problems.

Reply:
I prefer to use a base SI unit without an implied prefix for the units of parameters. The reason for this is that when I see `--genomeSize=3e9` it's clear what's going on. When I see `--genomeSize=3000` I don't know if the units are in bp, kbp, Mbp, or Gbp without reading the documentation. Additionally, when specifying the size of small genomes, such as the human mitochondrial genome, I found `--genomeSize=0.016569` to be rather awkward and confusing compared to `--genomeSize=16569`.

Reply:
For consistency though and to prevent trouble with `--merge-params` I'll change the parameter name to `genomeSizeBp`.